### PR TITLE
Version up to 1.4.7

### PR DESCRIPTION
**Pull request by babenek**
README.md was updated

## Description

- Verion was increased
- Readme was updated

## How has this been tested?

- [x] Check after release

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,7 +31,21 @@
 
 ## Introduction
 
-CredSweeper is a tool to detect credentials in any directories or files. CredSweeper could help users to detect unwanted exposure of credentials  (such as personal information, token, passwords, api keys and etc) in advance. By scanning lines, filtering, and using AI model as option, CredSweeper reports lines with possible credentials, where the line is, and expected type of the credential as a result.
+CredSweeper is a tool to detect credentials in any directories or files
```
Comment (csh519) [inline README.md:34]:
```suggestion
CredSweeper is a tool to detect credentials in any directories or files.
```

File: README.md [hunk 2]
```diff
@@ -31,7 +31,21 @@
...
+CredSweeper reports lines with possible credentials, where the line is,
+and expected type of the credential as a result.
+CredSweeper has experimental option ``--depth`` to scan files with predilection:
+- supported containers (tar, zip, gzip, bzip)
+- base64 encoded data
+- represent text (xml, html, json, yaml etc.) as structure
+- parse python source files with builtin ast engine
+
+Reported line numbers may be not actual in report but "info" field may help to understand
+how the credential was found.
```
Comment (csh519) [inline README.md:47]: IMO introducing `--depth` option from here seems inappropriate..
`--depth` option should be introduced in other document.
Or you can create experimental section for experimental options or functions like this and write it there.

Comment (babenek) [reply README.md]: The description was moved to guide.rst

**Review by csh519 — APPROVED** [review]
Thank you for updating.

LGTM 👍

File: docs/source/guide.rst [hunk 3]
```diff
@@ -67,6 +67,15 @@ Get all argument list:
     However, use of `--api_validation` is not recommended at the moment as its influence on False Positive/False Negative alerts are not validated yet.
     Moreover, it might result in a ddos related ban from corresponding APIs if number of requests is too high.
 
+.. note::
+    CredSweeper has experimental option `--depth` to scan files with predilection:
+        - supported containers (tar, zip, gzip, bzip2)
+        - base64 encoded data
+        - represent text (xml, json, yaml etc.) as structure and combine key with value
```
Comment (kmnls) [inline docs/source/guide.rst:74]:
```suggestion
        - represent text (xml, json, yaml etc.) as a structure and combine keys with values before analysis
```

**Review by iuriimet — DISMISSED** [review]

File: docs/source/guide.rst [hunk 4]
```diff
@@ -67,6 +67,15 @@ Get all argument list:
     However, use of `--api_validation` is not recommended at the moment as its influence on False Positive/False Negative alerts are not validated yet.
     Moreover, it might result in a ddos related ban from corresponding APIs if number of requests is too high.
 
+.. note::
+    CredSweeper has experimental option `--depth` to scan files with predilection:
```
Comment (kmnls) [inline docs/source/guide.rst:71]:
```suggestion
    CredSweeper has experimental option `--depth` to scan files when taking into account a knowledge about data formats:
```

**Review by kmnls — APPROVED** [review]

**Review by iuriimet — APPROVED** [review]

**Pull request merged**